.github/workflows/issue_labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Inspired by: https://github.com/stefanbuck/ristorante
+# See: https://stefanbuck.com/blog/codeless-contributions-with-github-issue-forms
+---
+
+name: Issue labeler
+on:
+  issues:
+    types: [ opened ]
+
+jobs:
+  label-component:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        template: [ bug_report.yml, feature_request.yml ]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Parse issue form
+        uses: stefanbuck/github-issue-praser@v2
```

Comment:
Looks like it should be `github-issue-parser`. It should also be pinned to SHA1 by analogy with the other actions.

Reply:
Yes! And I'm a bit confused how is possible that it works, but it looks like repository was originally called `github-issue-praser`.

Reply:
> @jamacku to figure out how it works in general I opened [evverx#115](https://github.com/evverx/systemd/issues/115). According to the logs GitHub actually downloaded and ran `github-issue-praser`

@evverx Hmm, are you sure that you have the same settings as the main systemd repo? On my fork it worked fine: https://github.com/jamacku/systemd/issues/3

> It works because it was renamed and GitHub redirects that to `github-issue-parser` but if `github-issue-praser` was created it would no longer be redirected to `github-issue-parser` and it's generally an example of how typosquatting on GH works. Are you 100% sure that action is trustworthy?

Hmm, that is interesting. I don't know the author of the action, but he seems to be legitimate and the action used quite a bit in my opinion. There are multiple actions for parsing issue forms and I chose this one based on output it generates and its usage. Also I found it interesting that one of the contributors is Gregor Martynus who works on octokit and github actions and apps related things.

Reply:
> are you sure that you have the same settings as the main systemd repo?

I was sure last year :-) Not so much now. According to the logs the token permissions were the same.

>  There are multiple actions for parsing issue forms and I chose this one based on output it generates and its usage. Also I found it interesting that one of the contributors is Gregor Martynus who works on octokit and github actions and apps related things.

Hopefully it was just a typo then.